### ISSUE TITLE
Verification Email

### ISSUE THREAD

**Issue by SteveUNSW** [RocketChat/Rocket.Chat#13109]
Version 0.73.2 for RocketChat breaks verification emails.

Steve

**Comment by MarcosSpessatto** [comment]
Hello @SteveUNSW can you please provide some logs or more informations about it?

**Comment by SteveUNSW** [comment]
When clicking on the verification link, a 500 error occurs:

Exception while invoking method 'verifyEmail' Error: Can't call setUserId in a method after calling unblock     at MethodInvocation.setUserId (/app/bundle/programs/server/packages/ddp-common.js:367:13)     at AccountsServer._loginUser (/app/bundle/programs/server/packages/accounts-base.js:700:22)     at AccountsServer._attemptLogin (/app/bundle/programs/server/packages/accounts-base.js:744:56)     at AccountsServer._loginMethod (/app/bundle/programs/server/packages/accounts-base.js:762:17)     at MethodInvocation.verifyEmail (/app/bundle/programs/server/packages/accounts-password.js:1040:21)     at MethodInvocation.methodsMap.(anonymous function) (/app/bundle/programs/server/packages/rocketchat_lib.js:2696:36)     at MethodInvocation.methodMap.(anonymous function) (packages/rocketchat_monitoring.js:2731:30)     at maybeAuditArgumentChecks (/app/bundle/programs/server/packages/ddp-server.js:1877:12)     at DDP._CurrentMethodInvocation.withValue (/app/bundle/programs/server/packages/ddp-server.js:902:126)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1304:12)     at DDPServer._CurrentWriteFence.withValue (/app/bundle/programs/server/packages/ddp-server.js:902:98)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1304:12)     at Promise (/app/bundle/programs/server/packages/ddp-server.js:902:46)     at new Promise (<anonymous>:null:null)     at Session.method (/app/bundle/programs/server/packages/ddp-server.js:875:23)     at /app/bundle/programs/server/packages/ddp-server.js:75

**Comment by MarcosSpessatto** [comment]
@ggazzo any ideas?

**Comment by MartinSchoeler** [comment]
Closing this due lack of updates, if you experience this on the latest release, feel free to open a new issue.

Thanks!
